.github/workflows/quantitative.yaml

```diff
@@ -4,14 +4,20 @@ on:
   push:
     branches:
       - main
+    paths:
+      - 'rules/**'
+      - '.github/workflows/quantitative.yaml'
```

Comment:
The idea is to execute quantitative tests only if there are changes under  `rules/**` or the job itself